src/homekit.cs

```diff
@@ -1671,12 +1683,72 @@ interface HMNumberRange {
 		NSNumber Max { get; }
 	}
 
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAccessoryOwnershipToken {
+		[Export("initWithData:")]
+		IntPtr Constructor (NSData data);
+	}
+
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAddAccessoryRequest {
+		[Export ("home")] 
+		HMHome Home { get; }
+		
+		[Export ("accessoryCategory")]
+		HMAccessoryCategory AccessoryCategory { get; }
+		
+		[Export ("accessoryName")]
+		NSString AccessoryName { get; }
```

Comment:
why not `string` ?
we normally use `NSString` for constants that are defined by Apple (and which, sometimes, compare pointers and not the string content)